cmd/bridge/main.go

```diff
@@ -592,6 +593,24 @@ func main() {
 		},
 	}
 
+	if *fRedirectPort != "" {
```

Comment:
We should validate that the port is a number if set

Reply:
Will use `fs.Int()` for parsing the port number